typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "<%= moduleName %>",
+	"version": "0.0.0",
+	"description": "",
+	"license": "MIT",
+	"repository": "<%= githubUsername %>/<%= moduleName %>",
+	"author": {
+		"name": "<%= name %>",
+		"email": "<%= email %>",
+		"url": "<%= website %>"
+	},
+	"engines": {
+		"node": ">=6"
+	},
+	"scripts": {
+		"build": "del dist && tsc",
```

Comment:
We should open an issue on TypeScript for `tsc` to get a `--cleanup` flag or something so we don't need `del dist`.

By "we", I'm hoping you can do it.

Reply:
As of reading through a bunch of TypeScript Issues:

They don't like to delete a folder completely (good summary https://github.com/Microsoft/TypeScript/issues/13722) but deleting output files when the source files are gone is an ongoing Issue (https://github.com/Microsoft/TypeScript/issues/16057).
Sadly this issue is not planned for one of the next releases (attached as a Milestones).